wcs.js

```diff
@@ -82,12 +82,15 @@ WebComponentShards.prototype = {
   _synthesizeImport: function _synthesizeImport() {
     return this._getCommonDeps().then(function(commonDeps) {
       var synthetic = '';
+      var workdir = url.resolve(this.root, this.workdir);
+      var importPath = url.resolve(workdir, this.shared_import);
+      var walk = path.relative(importPath, workdir);
```

Comment:
The variable names here in general are kinda tricky to read, and a comment at the top of what we're doing would go a long way. What do you think of this?

```javascript
      /** Generate the file of shared imports. */
      var output = '';
      var workdirPath = url.resolve(this.root, this.workdir);
      var outputPath = url.resolve(workdirPath, this.shared_import);
      var baseUrl = path.relative(outputPath, workdirPath);
      for (var dep in commonDeps) {
        output += '<link rel="import" href="' + baseUrl + '/' + commonDeps[dep] + '">\n';
      }
      var outDir = path.dirname(outputPath);
      mkdirp.sync(outDir);
      var fd = fs.openSync(outputPath, 'w');
      fs.writeSync(fd, output);
      return commonDeps;
```